.gitignore

```diff
@@ -20,6 +20,7 @@ latex/
 *.pdf
 *.tag
 .DS_Store
+locm3.sublime-workspace
```

Comment:
ahh, gotcha.  I missed the -workspace vs -project distinction.  my bad.  I'll pull this bit in too.